### PR TITLE
SABnzbd 3.4.2

### PR DESCRIPTION
**Pull request by Safihre**
- [x] Tested on DSM 6 `88f6281`, since that's the only one I own.

### CODE REVIEW

**Comment by Safihre** [comment]
@th0ma7 It now builds with your code!
Could you review and publish the update?

File: spk/sabnzbd/Makefile [hunk 1]
```diff
@@ -1,9 +1,9 @@
 SPK_NAME = sabnzbd
-SPK_VERS = 3.3.1
-SPK_REV = 49
+SPK_VERS = 3.4.2
+SPK_REV = 50
 SPK_ICON = src/sabnzbd.png
 
-BUILD_DEPENDS = cross/python38 cross/setuptools cross/pip cross/wheel
+BUILD_DEPENDS = cross/python38 cross/setuptools cross/pip cross/wheel cross/poetry
```
Comment (th0ma7) [inline spk/sabnzbd/Makefile:6]: Poetry is needed?  Out of curiosity, why?

**Comment by Safihre** [comment]
@th0ma7 or @publicarray could you merge and publish?

**Review by th0ma7 — APPROVED** [review]
LGTM

**Comment by th0ma7** [comment]
@Safihre now published.  Can you double-check all looks good?

**Pull request merged**